From 8d613d45f6fc0f9464bc5f6e5893b5945500c286 Mon Sep 17 00:00:00 2001
From: Leah Rowe <leah@libreboot.org>
Date: Sun, 13 Mar 2022 18:48:29 +0000
Subject: [PATCH 1/1] specifically call python3 in me_cleaner

it was referencing "python" which is ambiguous. this causes
problems on some setups
---
 me_cleaner.py | 2 +-
 setup.py      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/me_cleaner.py b/me_cleaner.py
index c2adf0e..d74b16b 100755
--- a/me_cleaner.py
+++ b/me_cleaner.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 
 # me_cleaner - Tool for partial deblobbing of Intel ME/TXE firmware images
 # Copyright (C) 2016-2018 Nicola Corna <nicola@corna.info>
diff --git a/setup.py b/setup.py
index 9928e5d..f04e336 100755
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 
 from setuptools import setup
 
-- 
2.25.1

